doc/lua_api.txt

```diff
@@ -6258,16 +6258,16 @@ object you are working with still exists.
     * radians: Angle from looking forward, where positive is downwards.
 * `set_look_horizontal(radians)`: sets look yaw
     * radians: Angle from the +z direction, where positive is counter-clockwise.
-* `get_look_pitch()`: pitch in radians - Deprecated as broken. Use
+* `get_look_pitch()`: **DEPRECATED as BROKEN** pitch in radians. Use
   `get_look_vertical`.
     * Angle ranges between -pi/2 and pi/2, which are straight down and up
       respectively.
-* `get_look_yaw()`: yaw in radians - Deprecated as broken. Use
+* `get_look_yaw()`: **DEPRECATED as BROKEN** yaw in radians. Use
   `get_look_horizontal`.
     * Angle is counter-clockwise from the +x direction.
-* `set_look_pitch(radians)`: sets look pitch - Deprecated. Use
+* `set_look_pitch(radians)`: **DEPRECATED** sets look pitch. Use
   `set_look_vertical`.
-* `set_look_yaw(radians)`: sets look yaw - Deprecated. Use
+* `set_look_yaw(radians)`: **DEPRECATED** sets look yaw. Use
```

Comment:
For the 4 line changes above:
Problem is the new order splits the message 'Deprecated, use xxx instead' by placing explanation text in the middle, so now it reads awkwardly.
To be honest i think these lines did not need changing, lower case to upper case is not necessary, people will not miss the deprecation message.
But, you are welcome to change the case as long as you keep the original order.

Reply:
My bad, I was trying to make it consistent with lines like `get_player_velocity` as in [here](https://github.com/minetest/minetest/blob/master/doc/lua_api.txt#L6248). Should I remove what it does as in `get_player_velocity` or do you prefer I simply revert my edits?

Reply:
Neither.
I think what it does should remain, so that people reading outdated code understand what that code does.
The use of upper case and asterisks is fine, so i do not request a revert.

So i suggest this form:
`get_look_pitch(): pitch in radians. **DEPRECATED as BROKEN**. Use get_look_vertical.`